database/schema.sql

```diff
@@ -179,8 +179,8 @@ ALTER TABLE dbo.DM_PERSONS ALTER COLUMN FULL_NAME TYPE varchar(255);
 
 create table dbo.SCCV_ALLOCATIONS
 (
-    MOSAIC_ID                   varchar(100) not null,      
-    FULL_NAME                   varchar(255) not null,
+    MOSAIC_ID                   varchar(100) not null,
+    FULL_NAME                   varchar(255),
```

Comment:
Please revert this file back to the previous version. Database updates need to be scripted so we can manage them properly. To remove the full_name not null constraint, you can add this below the table schema 
`alter table dbo.sccv_allocations alter column full_name drop not null;` so it can be run against existing table.

Reply:
Added